src/scope/scope.js

```diff
@@ -1010,6 +1009,10 @@ export default class Scope {
     return removeNils(componentsAndVersions);
   }
 
+  /**
+   * get consumerComponent by bitId only if is saved in the local scope. don't import from a remote scope.
```

Comment:
@davidfirst Do we really need this function here? isn't it's part of the consumer responsibility?

Reply:
That's right, we don't need. I actually had a note to remove it ("Get rid of scope.getConsumerComponent(), use consumer.loadComponentFromModel() instead.") along with two other notes that I forgot to follow. Will make these changes now.